recipe/meta.yaml

```diff
@@ -1,39 +1,83 @@
 {% set name = "evidently" %}
-{% set version = "0.3.3" %}
+{% set version = "0.4.39" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/evidentlyai/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 49f3f5a724e65369f33243732dacd48aad68e3aa0bd66e3a8e7784ed5f675b5f
+  sha256: c07a65a5c07138708328dbdf1e2b369f725039f714efb393b5631db7b57fa393
+  patches:
+    - 0001-patch-test-to-exclude-experimental-features.patch
 
 build:
   number: 0
   # umap-learn isn't available on s390x
   skip: True  # [py<37 or s390x]
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  # missing dynaconf for py312
+  skip: True  # [py>311]
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+    # removing experimental features that are dependent on missing chromadb
+    - rm -rf $SP_DIR/evidently/experimental
 
 requirements:
+  build:
+    - patch     # [not win]
+    - m2-patch  # [win]
   host:
     - pip
     - python
     - setuptools
     - wheel
   run:
-    - nltk >=3.6.7
-    - numpy >=1.19.5
-    - pandas >=1.3.5
-    - plotly >=5.5.0
-    - pydantic >=1.9,<2
     - python
-    - pyyaml >=5.1
-    - requests >=2.19.0
-    - scikit-learn >=0.24.0
-    - scipy >=1.5.4
+    - plotly >=5.10.0
     - statsmodels >=0.12.2
-    - umap-learn >=0.5.3
+    - scikit-learn >=1.0.1
+    - pandas >=1.3.5
+    - numpy >=1.22.0,<2.1
+    - nltk >=3.6.7
+    - scipy >=1.10.0
+    - requests >=2.32.0
+    - PyYAML >=5.4
+    - pydantic >=1.10.13
+    - litestar >=2.8.3
+    - typing_inspect >=0.9.0
+    - uvicorn >=0.22.0
+    - watchdog >=3.0.0
+    - typer >=0.3
+    - rich >=13
+    - iterative-telemetry >=0.0.5
+    - dynaconf >=3.2.4
+    - certifi >=2024.7.4
+    - urllib3 >=1.26.19
+    - fsspec >=2024.6.1
+    - ujson >=5.4.0
+    - deprecation >=2.1.0
+    - uuid6 >=2024.7.10
+    - cryptography >=43.0.1
+    # transformers would be optional but it is included in BERTScore_descriptor.py
+    - transformers >=4.39.3
+  # from https://github.com/evidentlyai/evidently/blob/cd6cbfba3f0c0ebdb7494e53941d954272f451fa/requirements.min.txt#L30
+  run_constrained:
+    - openai >=1.16.2
+    - evaluate >=0.4.1
+    - sentence-transformers >=2.7.0
+    - chromadb >=0.4.0
+    - blas=*=openblas  # [osx and x86_64]
```

Comment:
Should `pytorch` and `accelerate` be here also? See https://github.com/evidentlyai/evidently/blob/v0.4.39/setup.py#L104 and https://github.com/huggingface/transformers/blob/v4.45.2/setup.py#L280

Reply:
And these optional dependencies https://github.com/evidentlyai/evidently/blob/v0.4.39/setup.py#L108C20-L111

Reply:
I could add them, yes.

Reply:
actually there is no version constraint on `pytorch`. I added `accelerate` and a few more optional dependencies.